src/services/findAllReferences.ts

```diff
@@ -709,10 +709,26 @@ namespace ts.FindAllReferences.Core {
         return references.length ? [{ definition: { type: DefinitionKind.Symbol, symbol }, references }] : emptyArray;
     }
 
+    /** As in a `readonly prop: any` or `constructor(readonly prop: any)`, not a `readonly any[]`. */
+    function isMemberModifyingReadonlyKeyword(node: Node): boolean {

```

Comment:
My internalization of @sandersn PR reviews is starting to develop, and I could already hear the voice in the back of my head saying that he wasn’t sold on the name. I’m not either, but I struggled to come up with something more accurate and pithy. In `readonly string[]`, it’s not strictly a “modifier” but an “operator,” so technically I could just mention “modifier” without “member” as a disambiguation, but I think `readonly` in `readonly string[]` kinda _looks_ like a modifier... I went back and forth for 10 minutes or so before deciding I needed to move on with my life.

Reply:
`isReadonlyKeywordOnProperty`

Reply:
isReadonlyModifierForMemberDeclaration